client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "0.3.1",
+  "version": "0.3.2",
```

Comment:
The package-lock.json file needs to be updated as well.

If you did a `npm i` after updating the value above, it will automatically update the lock file.